Shifud/plugincommon/basewebscanner.go

```diff
@@ -0,0 +1,40 @@
+package plugincommon
+
+import (
+	"example.com/Shifud/shifud"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+)
+
+type ProtocolHandler func(ip string, port int, timeout time.Duration) bool
+
+func WebScanner(startPort, endPort int, timeout time.Duration, handler ProtocolHandler) []shifud.DeviceConfig {
+	var devices []shifud.DeviceConfig
+	var wg sync.WaitGroup
+
+	log.Println("Launching our little scanner minions! 🚀")
+	for i := 1; i <= 254; i++ {
+		ip := fmt.Sprintf("192.168.1.%d", i)
```

Comment:
make 192.168.1. as a const
BTW, one question, would the network always start with 192.168.1?